src/chains/hemi-sepolia.ts

```diff
@@ -43,6 +43,11 @@ export const hemiSepolia = defineChain({
         address: "0xc94b1BEe63A3e101FE5F71C80F912b4F4b055925",
       },
     },
+    l2Bridge: {
+      [sourceId]: {
+        address: "0x4200000000000000000000000000000000000010",
+      },
+    },
```

Comment:
Need to confirm with @max-sanchez  if this is correct or if the address is the same for every chain, so we would just do:

```suggestion
    l2Bridge: {
        address: "0x4200000000000000000000000000000000000010",
    },
```

Reply:
Max confirmed with me it is better to save it indexed by `sourceId`